### DISCUSSION TITLE
Bypassing JVM index lookup when using coursier/cs

### DISCUSSION THREAD

**Discussion by DamianReeves** [coursier/coursier#2899] [Q&A]
We are trying to use coursier in our CI within my company and we are hitting issues where the coursier commands are reaching out to download the JVM Index which seems to be blocked in our org.
Are there any flags or environment variables we can pass in to ensure that this reachout doesn't happen?

**Comment by tgodzik** [comment]
Maybe using `--jvm-index string?     Location of the JVM index that you'd like to use` would help here?
